python/pyarrow/tests/test_flight.py

```diff
@@ -506,6 +505,162 @@ def get_token(self):
         return self.token
 
 
+class NoopAuthHandler(ServerAuthHandler):
+    """A no-op auth handler."""
+
+    def authenticate(self, outgoing, incoming):
+        """Do nothing."""
+
+    def is_valid(self, token):
+        """
+        Returning an empty string.
+        Returning None causes Type error.
+        """
+        return ""
+
+
+def case_insensitive_header_lookup(headers, lookup_key):
```

Comment:
This is specifically to extract an authentication header based n the exception it raises, let's make sure the name reflects that.

Reply:
I modified the function to not raise an exception when the header is not found. The method is used in a test not related to authentication as well.